src/backend/distributed/commands/citus_version.c

```diff
@@ -0,0 +1,36 @@
+/*-------------------------------------------------------------------------
+ *
+ * citus_version.c
+ *	  Routines related to the version of the Citus binaries
+ *
+ * Copyright (c) 2012-2016, Citus Data, Inc.
+ *
+ *-------------------------------------------------------------------------
+ */
+
+#include "postgres.h"
+
+#include "fmgr.h"
+
+#include "citus_version.h"
+#include "utils/builtins.h"
+
+
+/* exports for SQL callable functions */
+PG_FUNCTION_INFO_V1(citus_running_version);
+
+#if !defined(CITUS_BINARY_VERSION)
```

Comment:
I'd _really_ like the binary version to contain the patch number. Even if it's not needed for all of our comparisons, it will be nice to have down the line. For the same reason I think it might be nice to have some sort of `CITUS_BUILD_INFO` field that I can populate in our build system (with `FINAL` or the nightly SHA, or whatever).

The patch version is probably what I'd like most, though, since it would be nice to be able to have users be able to answer "am I actually running the latest patch with the bugfix I want" from within `psql`. Sure, this will require hardcoding the major/minor/patch versions somewhere, but it's good enough for PostgreSQL, right?